jib-core/src/test/java/com/google/cloud/tools/jib/plugins/common/ConfigurationPropertyValidatorTest.java

```diff
@@ -69,58 +70,82 @@ public void testCheckHttpTimeoutSystemProperty_negativeValue() {
 
   @Test
   public void testGetImageAuth() {
-    Mockito.when(mockAuth.getUsernamePropertyDescriptor()).thenReturn("user");
-    Mockito.when(mockAuth.getPasswordPropertyDescriptor()).thenReturn("pass");
-    Mockito.when(mockAuth.getUsername()).thenReturn("vwxyz");
-    Mockito.when(mockAuth.getPassword()).thenReturn("98765");
-
     // System properties set
-    System.setProperty("jib.test.auth.user", "abcde");
-    System.setProperty("jib.test.auth.pass", "12345");
+    System.setProperty("jib.test.auth.username", "abcde");
+    System.setProperty("jib.test.auth.password", "12345");
     Authorization expected = Authorizations.withBasicCredentials("abcde", "12345");
     Authorization actual =
-        ConfigurationPropertyValidator.getImageAuth(
-            mockLogger, "jib.test.auth.user", "jib.test.auth.pass", mockAuth);
+        ConfigurationPropertyValidator.newMavenPropertyValidator(mockLogger)
+            .getImageAuth("test", null, null);
     Assert.assertNotNull(actual);
     Assert.assertEquals(expected.toString(), actual.toString());
 
     // Auth set in configuration
-    System.clearProperty("jib.test.auth.user");
-    System.clearProperty("jib.test.auth.pass");
+    System.clearProperty("jib.test.auth.username");
```

Comment:
Huh, were we testing wrong properties?

Reply:
Nope, originally we used to pass the system property name into the method (so I used a different one in the test), but now it's just hard coded into the method.